src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -317,7 +326,10 @@
        "/workspaces"
        {:on-done (fn [{:keys [success? xhr]}]
                    (if success?
-                     (let [workspaces (utils/parse-json-string (.-responseText xhr))]
+                     (let [raw-workspaces (utils/parse-json-string (.-responseText xhr))
+                           ;; The following is just to make the icons work with the
+                           ;; current live data, which uses this missing field
+                           workspaces (map #(assoc % "status" "Complete") raw-workspaces)]
```

Comment:
I think this file is getting too long. Maybe a TODO at the top to refactor into multiple files?